python/grizzly/setup.py

```diff
@@ -0,0 +1,42 @@
+import os
+import platform
+import shutil
+import subprocess
+import sys
+
+from setuptools import setup, Distribution
+import setuptools.command.build_ext as _build_ext
+from setuptools.command.install import install
+
+system = platform.system()
+if system == 'Linux':
+    numpy_convertor = "numpy_weld_convertor.so"
+elif system == 'Windows':
+    numpy_convertor = "numpy_weld_convertor.dll"
+elif system == 'Darwin':
+    numpy_convertor = "numpy_weld_convertor.dylib"
+else:
+    raise OSError("Unsupported platform {}", system)
+
+class Install(install):
+    def run(self):
+        install.run(self)
+        python_executable = sys.executable
+        protoc_command = ["make -C " + self.install_lib + "grizzly/ EXEC=" + python_executable]
```

Comment:
Should we invoke make clean before this?

Reply:
Yeah, that sounds like a good idea